### DISCUSSION TITLE
Enemies for horror mode:

### DISCUSSION THREAD

**Discussion by deltasolidsnake** [WizardMaster38/Infernum-Ad-Infinitum-To-Hell-and-Beyond#11] [Ideas]
The following individual comments will be the enemy suggestions.

**Comment by deltasolidsnake** [comment]
Keep in mind, for the horror game mode, the true horror of Hell will be capitalized on. Most enemies will be grotesque and unnerving.

**Reply by WizardMaster38** [reply]
Go ahead and describe them

**Comment by deltasolidsnake** [comment]
![abhoth-gibbering-mouther-small](https://user-images.githubusercontent.com/88522032/128455710-1ba46fc9-121a-41da-bcfe-df2c56e1dcbb.jpg)
**Corrupted slime**
The corrupted slime is a rather harmless slime that has been infested with a bacterium that makes the slime malicious, think of it like rabies. But, much more terrifying as the slime picks up objects from its surroundings, creating structures like legs, teeth, and weapons. 
The slime would have a reddish black color, signifying the growth of bacteria.

(The infection is mutating them, causing the unusual behavior.)
_**Enemy category: Undead**_

**Reply by WizardMaster38** [reply]
Scary

**Reply by deltasolidsnake** [reply]
A lot of these are going to be inspired from Dead Space, like this one.

**Reply by WizardMaster38** [reply]
Good idea, already have slimes anyway, so will be adding this!

**Comment by deltasolidsnake** [comment]
_**Wendigo**_
The Wendigo is a creature that stalks and hunts its prey through mimicry, stealth, and intimidation. It is very intelligent and cannot be outsmarted easily. The creature does not feed on flesh, but rather the soul of its kill. This will be a boss type enemy.


_**Enemy category: Paranormal**_
![128fdd9dcc6a751beb6fc192a5473dd1](https://user-images.githubusercontent.com/88522032/128455439-e4b5d754-fbde-4eb5-9f21-369cebdb3b7f.jpg)

**Reply by WizardMaster38** [reply]
Neat boss!

**Comment by deltasolidsnake** [comment]
![2959e3a717bb9b3cb5c7296bc195579a--monster-design-monter](https://user-images.githubusercontent.com/88522032/128456173-6c16c5de-6eeb-435b-b503-6f9baa28feca.jpg)
_**Lure**_

Lures are demons that crawl on sealings and walls using their long bioluminescent tongues as lures, usually by waving it around to appear as a firefly. They would be encountered in caves ad forests.

_**Enemy category: Demon**_

**Reply by WizardMaster38** [reply]
Enemy category?

**Reply by deltasolidsnake** [reply]
oops

**Reply by WizardMaster38** [reply]
Easy enough to convey through text, could be used for ambushes. Will be added!

**Comment by deltasolidsnake** [comment]
_**Fallen Angel**_
Once the most beautiful in all the universe, now corrupted, turned inglorious by their own actions.
These are heavy type enemies, used as minibosses or tanks.
When killed, a beam of an unknown substance and light burst from the angel at light speed towards the sky. I wonder where it's going?
_**Enemy category: Demon**_

![a4ac03b9feef70493a4defa08af9a2c8](https://user-images.githubusercontent.com/88522032/128458136-6f2f12dd-c410-4ff0-af3e-44e085d929ee.jpg)
![160aad20578ae3511d046a13f64ed6bb](https://user-images.githubusercontent.com/88522032/128458140-3e0683bf-7bed-44f0-abcb-df2f9844353a.jpg)

**Reply by WizardMaster38** [reply]
Good idea, seems hard to get across through text though. Gonna give it some more thought, it's a solid maybe for now.

**Comment by deltasolidsnake** [comment]
_**Amalgam/Amalgamate**_

The Amalgamate is a collection of biomass formed into one horrible being, it only appears in areas of extreme undead activity.
Causes psychological debuffs to the characters. It is used as a boss.

_**Enemy category: Undead**_
![f14ff3e96012228897e7dfe953bd8383](https://user-images.githubusercontent.com/88522032/128458505-e29edcea-5066-4574-9877-b9c5ce19811a.jpg)

**Reply by WizardMaster38** [reply]
Seems too scary and hard to pinpoint accurately through text

**Reply by deltasolidsnake** [reply]
fair point.

**Comment by deltasolidsnake** [comment]
_**Eshgolem**_

The Eshgolem is an enemy that is tanky and defends valuable areas like objectives. They are weak to ranged attacks and magic.
(Esh meaning fire in Hebrew, and golem referring to the golem in Jewish folklore.)
![Lava_Demon](https://user-images.githubusercontent.com/88522032/128514261-7afb54b7-134b-404c-b4fc-06de1365928e.jpg)

**Reply by WizardMaster38** [reply]
Could use some work, solid maybe.

**Comment by deltasolidsnake** [comment]
The representation through text is not a big deal to me, I'm just giving reference.
